examples/Benchmark.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python3
```

Comment:
Could you rename this file `examples/benchmark.py`? I forgot to mention in the last review, but the codebase standard is to have lower-case names.